go.graphics.swing/src/go/graphics/swing/AreaContainer.java

```diff
@@ -35,6 +28,13 @@
 import com.jogamp.opengl.awt.GLJPanel;
 import com.jogamp.opengl.glu.GLU;
 
+import go.graphics.RedrawListener;
+import go.graphics.area.Area;
+import go.graphics.event.GOEvent;
+import go.graphics.event.GOEventHandlerProvider;
+import go.graphics.swing.event.swingInterpreter.GOSwingEventConverter;
+import go.graphics.swing.opengl.JOGLDrawContext;
+
```

Comment:
I may did a change and undo it, while I was trying.
The question is more, why is the order changed?
I use Eclipse "Mars Release (4.5.0)" with the formatting settings imported from you, and the Organize imports activated as written in the Development Guidelines here: https://github.com/jsettlers/settlers-remake#eclipse